cmd/openshift-install/create.go

```diff
@@ -350,6 +350,16 @@ func waitForBootstrapComplete(ctx context.Context, config *rest.Config) *cluster
 	discovery := client.Discovery()
 
 	apiTimeout := 20 * time.Minute
+
+	// Wait longer for baremetal, due to length of time it takes to boot
+	if assetStore, err := assetstore.NewStore(rootOpts.dir); err == nil {
```

Comment:
Same code is used in multiple places, should a function be used?

Reply:
Identical logic here https://github.com/openshift/installer/blob/master/cmd/openshift-install/create.go#L449-L455 while waiting for the cluster to be initialized.